handbook/people-ops/onboarding/index.md

```diff
@@ -18,6 +18,50 @@ At Sourcegraph, we don't think of teammates as resources — we think of them as
 
 Never assume that a problem is somebody else's to fix!
 
+## Onboarding standards and success
+
+### Onboarding expectations
+This is a basic breakdown of our expectations for providing a smooth onboarding process:
+
+- New hire has the tools and resources they need to navigate their day to day (accounts, software...)
+- New hire has the necessary support from their peers, manager, and People team (regular 1-1s, shadowing...)
+- New hire knows where to find answers to their questions 
+- New hire experiences no logistical challenges (payroll, expenses, benefits, setting up accounts or other systems...)
+- The onboarding checklist is manageable and easy to navigate by new hire
```

Comment:
Do we need a bullet that covers something like "New teammate has the necessary understanding of Sourcegraph's products, customer value propositions, internal processes, values, and anything needed so they can make an impact" or similar? 

That phrasing is wordy, but the idea here would be, e.g., that providing product training to the sales team doesn't seem to be covered by the bullets above. Or do you think it would fall under one of the ones above?

Reply:
Good point– I reworded this a bit and added a bullet to address this!